.ci/jenkins/dsl/jobs.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+* This file is describing all the Jenkins jobs in the DSL format (see https://plugins.jenkins.io/job-dsl/)
+* needed by the Kogito pipelines.
+*
+* The main part of Jenkins job generation is defined into the https://github.com/kiegroup/kogito-pipelines repository.
+*
+* This file is making use of shared libraries defined in
+* https://github.com/kiegroup/kogito-pipelines/tree/main/dsl/seed/src/main/groovy/org/kie/jenkins/jobdsl.
+*/
+
+import org.kie.jenkins.jobdsl.model.JenkinsFolder
+import org.kie.jenkins.jobdsl.model.JobType
+import org.kie.jenkins.jobdsl.utils.EnvUtils
+import org.kie.jenkins.jobdsl.utils.JobParamsUtils
+import org.kie.jenkins.jobdsl.KogitoJobTemplate
+import org.kie.jenkins.jobdsl.KogitoJobUtils
+import org.kie.jenkins.jobdsl.Utils
+
+jenkins_path = '.ci/jenkins'
+
+///////////////////////////////////////////////////////////////////////////////////////////
+// kie-benchmarks jobs
+///////////////////////////////////////////////////////////////////////////////////////////
+
+// update drools/optaplanner version in kie-benchmarks
+setupDroolsUpdateJob()
+setupOptaPlannerUpdateJob()
+
+void setupDroolsUpdateJob() {
+    def jobParams = JobParamsUtils.getBasicJobParams(this, 'kie-benchmarks-update-drools', JobType.TOOLS, "${jenkins_path}/Jenkinsfile.bump-up-version", 'Drools bump up version')
+    JobParamsUtils.setupJobParamsDefaultMavenConfiguration(this, jobParams)
+    jobParams.env.putAll([
+        JENKINS_EMAIL_CREDS_ID: "${JENKINS_EMAIL_CREDS_ID}",
+
+        GIT_BRANCH_NAME: "${GIT_BRANCH}",
+
+        IS_MAIN_BRANCH: "${Utils.isMainBranch(this)}",
+
+        REPO_NAME: 'drools',
```

Comment:
please use another name. `REPO_NAME` is used to identify the repository against which you run. Here it is `kie-benchmarks`